test_scripts/Smoke/Resumption/ATF_Resumption_3rd_ignition_cycle.lua

```diff
@@ -106,15 +91,31 @@ end
 commonFunctions:newTestCasesGroup("SDL should perform data resumption application is registered within 3 ign cycles")
 
 function Test.IGNITION_OFF()
-  IGNITION_OFF()
+  Test.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
```

Comment:
@dboltovskyi Please use `self` instead direct module variable `Test` in member functions.

Reply:
@aderiabin Test was used in original version of the script. Do we really need to remove it?

Reply:
@aderiabin Please check d5a3629